### PR TITLE
Add large benchmark directory

### PR DESCRIPTION
**Pull request by Tmonster**
So that we can test tpch at sf100 weekly on a self hosted GitHub Actions runner without having to manage a new benchmark runner.

### CODE REVIEW

**Comment by Mytherin** [comment]
Thanks! LGTM

File: extension/tpch/dbgen/answers/sf100/q01.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+l_returnflag|l_linestatus|sum_qty|sum_base_price|sum_disc_price|sum_charge|avg_qty|avg_price|avg_disc|count_order
```
Comment (alamb) [inline extension/tpch/dbgen/answers/sf100/q01.csv:1]: 👋  hi. Where did you get these expected answers? Are they from DuckDB itself or did you generate / compute them somewhere else?

Context is we would like to offer something similar in a Rust generator, and it isn't clear to us how to compute the expected answers for different scale factors:
- https://github.com/clflushopt/tpchgen-rs/issues/103

Comment (Tmonster) [reply extension/tpch/dbgen/answers/sf100/q01.csv:1]: I think we computed them ourselves and then double checked the results by comparing with other db systems (e.g. postgres)

Comment (alamb) [reply extension/tpch/dbgen/answers/sf100/q01.csv:1]: Thank you @Tmonster 🙏

**Pull request merged**